### ISSUE TITLE
Method not found CopyTo

### ISSUE THREAD

**Issue by Hi-ImKyle** [BepInEx/BepInEx#97]
**Describe the bug**
BepInEx doesn't launch, instead outputs a exception in the log.

**To Reproduce**
Run The Escapists 2 with BepInEx.

**Expected behavior**
Should just run fine.

**Screenshots and logs**
```
BepInEx 4.1.2.0 - TheEscapists2
[Message] Preloader started
[Fatal] Could not run preloader!
[Fatal] System.MissingMethodException: Method not found: 'System.IO.Stream.CopyTo'.
  at Mono.Cecil.ModuleDefinition.ReadModule (System.String fileName) [0x00000] in <filename unknown>:0 
  at Mono.Cecil.AssemblyDefinition.ReadAssembly (System.String fileName) [0x00000] in <filename unknown>:0 
  at BepInEx.Bootstrap.AssemblyPatcher.PatchAll (System.String directory, IDictionary`2 patcherMethodDictionary, IEnumerable`1 initializers, IEnumerable`1 finalizers) [0x00000] in <filename unknown>:0 
  at BepInEx.Bootstrap.Preloader.Run () [0x00000] in <filename unknown>:0
```

**Desktop (please complete the following information):**
 - Game and game version [e.g. COM3D2 1.15]: The Escapists 2 `Steam_Windows_Release_1.1.10.566064`
 - BepInEx version [e.g. 4.0]: 4.1.2.0

**Additional context**
~N/A~

**Comment by bbepis** [comment]
Hi,
Please try this again with BepInEx 5.0 RC1. 4.X is no longer a supported version and a significant amount of changes, both features and stability improvements have been added since then.

**Comment by ManlyMarco** [comment]
Closing because of inactivity. @Hi-ImKyle If the issue is not fixed in the latest release (5.0) please reopen this issue and provide the new log.
